### DISCUSSION TITLE
Yi 1080p Home QFUS  firmwareversion 12.0.35.8

### DISCUSSION THREAD

**Discussion by icho40** [roleoroleo/yi-hack-Allwinner-v2#748] [Q&A]
My camera is supported according to the list.
I have downloaded the files for y291ga accordingly, unpacked them and copied them to the SD card.
Now the blue LED on the camera flashes continuously.
After waiting 15 minutes, I restarted the camera.
Same result.
What can I do?

**Comment by roleoroleo** [comment]
Extract the sd card and check if the cam starts normally without it.
Insert the sd card in your pc and check if a folder Factory.done exists.

**Reply by icho40** [reply]
I tried with wrong firmware first.
After extracting the sd card, the camera starts with yellow light blinking and "waiting to connect", but no wifi network is showing.
On the sd-card ist no folder "Factory.Done"
I pushed the reset button several times longer than 10 seconds, but nothing changed.

**Reply by roleoroleo** [reply]
> I tried with wrong firmware first.

Which fw?

**Reply by icho40** [reply]
I think, it was Yi 1080p Home  rootfs_y20 home_y20 from here:
https://github.com/shadow-1/yi-hack-v3/tree/master

**Reply by roleoroleo** [reply]
This shouldn't be a problem.
If you have a yellow light, try the unbrick procedure.

**Reply by icho40** [reply]
The yellow light flashes and the Voice says: waiting to connect.

> try the unbrick `procedure.

Do you mean this:
[https://github.com/roleoroleo/yi-hack-Allwinner-v2/tree/master/unbrick](url)

The problem is, i don't have a backup file.
Is there another way to unbrick?

**Reply by icho40** [reply]
If I can't do it any other way than with a backup, that's not so bad.
The camera wasn't expensive and it was available in a twin pack.
So I have a second one:
QFUSY (indoor)
FW: 12.0.35.8
What would be the right firmware for Yi-Hack?
I suppose it is y291ga

Thank you very much for your help and your work with Yi-Hack.

**Reply by roleoroleo** [reply]
Yes it's a y291ga.

If the cam is speaking, are you able to connect it using the app?

**Reply by icho40** [reply]
Yes to both questions.

**Reply by roleoroleo** [reply]
Please, try if the backup procedure works.
After the procedure you should find a copy of all mtdblocks inside the sd card.

And what about a telnet connection? Find the ip of the cam and try to connect with a telnet session.

**Reply by icho40** [reply]
If this is what you mean, I tried that.
https://github.com/roleoroleo/yi-hack-Allwinner-v2/wiki/Dump-your-backup-firmware-(SD-card)
Unfortunately it did not work.
There is no /Backup folder
I couldn't find the camera with nmap -sn.
It just keeps saying: waiting to connect
So telnet does not work

Thanks for your patience

**Reply by roleoroleo** [reply]
If the backup procedure doesn't work, there are just 2 options:
- the cam isn't Allwinner based
- the sd card is not properly recognized

Please, try another card.

**Reply by icho40** [reply]
On my 1st attempt the card was formatted incorrectly, sorry.
Now I have a backup file:
homever.txt
mtdblock0.bin
mtdblock1.bin
mtdblock2.bin
mtdblock3.bin
mtdblock4.bin
mtdblock5.bin
mtdblock6.bin
mtdblock7.bin

**Reply by roleoroleo** [reply]
If you want to open it, the label on the chip should be qg2101a.
Use a small card: 4 or 16 GB.

**Reply by icho40** [reply]
I opened it, the label matches.
Should I now perform the unbrick procedure?
Or should i try the Yi-hack again?

**Reply by roleoroleo** [reply]
If the backup worked, try the hack again.
It should work.

**Reply by icho40** [reply]
I tried the hack.
i copied y291ga_0.3.0-3, which had 2 directories, Factory and yi-hack 
and 1 file lower_half_init.sh on a fat32 formatted sd.
After reboot of the cam with inserted sd, there where
2 new directories; Backup and record
and a swapfile

But the cam still doesn't open a wifi.

Then i wanted to run the unbrick procedure, but
The unbrick-READ.md says:

Copy your partition (mtdblockX.bin) in the folder corresponding to your model.

According to
https://github.com/roleoroleo/yi-hack-Allwinner-v2#supported-cameras
i need y291ga, but at
https://github.com/roleoroleo/yi-hack-Allwinner-v2/tree/master/unbrick
is no y291ga

**Reply by roleoroleo** [reply]
Wait befor to apply the unbrick procedure.
Hack must work.
> i copied y291ga_0.3.0-3, which had 2 directories, Factory and yi-hack
> and 1 file lower_half_init.sh on a fat32 formatted sd.

Did you copy the compressed archive or the content?
The content of the sd card before inserting it should be:
```
Factory
yi-hack
lower_half_init.sh
```
After the hack the folder Factory should be renamed to Factory.done

**Reply by icho40** [reply]
I am terribly sorry.
I took out the sd card to see what was on it.
Then I had an important appointment and took the SD card with me.
I accidentally overwrote it there.
Now when I copy y291ga_0.3.0-3 to the SD card and put it in the camera, the yellow light stays off.

**Reply by icho40** [reply]
i had these files on the sd:
backup
Factory.done
lower_half_init.sh
record
swapfile
yi-hack

but now they are gone.

**Reply by roleoroleo** [reply]
It's not possible to damage the camera in this manner.
Please, try again from scratch.

**Reply by icho40** [reply]
Ok, now i have again these folders and files on the sd card:
backup
Factory.done
lower_half_init.sh
record
swapfile
yi-hack

Still now Wifi.
What now?

**Reply by roleoroleo** [reply]
Factory.done means that the hack is ok.
About wifi...
- are you sure the cam is not connected?
- did you check your router to see if the cam obtained an ip address?
- does the cam speak asking for a connection?
- does the yi app work?

**Reply by icho40** [reply]
> are you sure the cam is not connected?

yes
> did you check your router to see if the cam obtained an ip address?

yes, I even scanned my local network with nmap and couldn't find the cam
> does the cam speak asking for a connection?

The cam says:"Waitung to connect" and the yellow light is blinking.
> does the yi app work?

yes, my other Yi-home cam works fine.
![Yi-cam](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/8980118/64d25a1d-26af-45b4-8cab-eaf289109d73)

**Reply by icho40** [reply]
I managed to add the cam to the app.
Seems like it's working korrektly.
Thanks for your patience and your help.

Greetings from Ulm/germany
Willi

**Reply by roleoroleo** [reply]
Glad to hear you solved the problem.
Thank you for your donation.
